docs/getting-started/configuration.md

```diff
@@ -227,7 +226,7 @@ did in below example:
 integrations:
   github:
     - host: github.com
-      token: ghp_urtokendeinfewinfiwebfweb
+      token: $GITHUB_TOKEN
```

Comment:
This doesn't really end up adding the token though? Perhaps we could point to adding it as a production secret using this method, or for local development recommend an entry in `app-config.local.yaml`

Reply:
Hmm ok - let me take another pass. I think I misunderstood the note